Kitodo/src/main/java/org/kitodo/production/services/workflow/WorkflowControllerService.java

```diff
@@ -350,39 +350,51 @@ public void unassignTaskFromUser(Task task) throws DataException {
      *
      * @param comment as Comment object
      */
-    public void reportProblem(Comment comment) throws DataException {
+    public void reportProblem(Comment comment, TaskEditType taskEditType) throws DataException {
```

Comment:
As you change the method signature by adding a new parameter please adjust the Javadoc.